client/src/components/files/FileAggregateItem.vue

```diff
@@ -34,20 +35,22 @@
       </div>
     </ion-item>
     <ion-progress-bar
+      v-if="tab === FileOperationTabs.InProgress"
       class="progress-bar"
       type="indeterminate"
     />
   </div>
 </template>
 
 <script setup lang="ts">
-import { MsImage, Copy, FileImport, Move } from 'megashark-lib';
+import { MsImage, Copy, FileImport, Move, NoImportDone, NoImportError } from 'megashark-lib';
 import { IonIcon, IonItem, IonProgressBar, IonText, IonButton } from '@ionic/vue';
 import { closeCircle } from 'ionicons/icons';
-import { FileOperationDataType } from '@/services/fileOperationManager';
+import { FileOperationDataType, FileOperationTabs } from '@/services/fileOperationManager';
 
 const props = defineProps<{
   amount: number;
+  tab: FileOperationTabs;
```

Comment:
`Tabs` should only exist in the `FileOperationMenu`, it shouldn't be exported. The item has no need to know if it's displayed in a tab.

Beside the naming problem, I would just dupplicate the component. Have one for done items, one for errors, one for in progress. It would avoid those weird state switches, which are only going to get weirder once @fabienSvtr starts adding classes everywhere. It can be done using a `<slot />` to avoid too much copy/pasting.